### DISCUSSION TITLE
I can't set auto-reply in ProtonMail because the emails is from mailing lists

### DISCUSSION THREAD

**Discussion by JasonMoriarty** [simple-login/app#1165] [Feature Requests]
Is there any way to disable the feature? 
I set a simplelogin address as a recipient to auto reply when anyone sends to the simplelogin address. But it doesn't work because it has "List-Unsubscrib". I really want a button to disable the feature.

**Comment by nguyenkims** [comment]
@JasonMoriarty I'm not sure to understand how the "List-Unsubscrib" header interferes with auto-reply, can you elaborate more?

**Reply by JasonMoriarty** [reply]
@nguyenkims Auto reply on ProtonMail will not respond to automatically generated messages. Automatically generated messages include emails from mailing lists, emails sent via auto reply from a different email address, and messages sent from a noreply address.

**Reply by nguyenkims** [reply]
You can enable this new option so SimpleLogin won't automatically add this header.

<img width="1014" alt="image" src="https://user-images.githubusercontent.com/3167780/180049773-177bc884-5bfe-48cd-8ef5-10294049af0a.png">

**Reply by JasonMoriarty** [reply]
it works! but ProtonMail auto-reply still doesn't work...I may should contact their support. Thank you so much!
